examples/reference/widgets/ChatFeed.ipynb

```diff
@@ -258,14 +274,48 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "chat_feed.send(\"Are you a parrot?\", respond=False)"
+    "entry = chat_feed.send(\"Are you a parrot?\", respond=False)"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The easiest and most optimal way to stream output is through async generators.\n",
+    "The `ChatFeed` also support *async* `callback`s. In fact we recommend using *async* `callback`s whenever possible to keep your app fast and responsive."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from asyncio import sleep\n",
+    "\n",
+    "async def parrot_message(contents, user, instance):\n",
+    "    await sleep(1)\n",
+    "    return {\"value\": f\"No, {contents.lower()}\", \"user\": \"Parrot\", \"avatar\": \"🦜\"}\n",
+    "\n",
+    "chat_feed = pn.widgets.ChatFeed(callback=parrot_message, callback_user=\"Echo Bot\", callback_avatar=\"E\")\n",
+    "chat_feed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# TODO: IT'S LIKE THE its not sleeping but just displaying right away. The spinner is not displayed either. Andrew should look at this. `async` functions with `return` should also work.\n",
```

Comment:
It like this example with an `async` function with a `return` statement does not work correctly. Its like the `await sleep` is never run. And the `LoadingIndicator` is not triggered either.

Would you take a look @ahuang11 ? Thanks.

Reply:
I'm a bit confused with this request. Since `respond=False`, the callback will not execute.